app/src/main/res/values/strings.xml

```diff
@@ -19,4 +19,18 @@
     <!-- Misc -->
     <string name="dev_options_button">Dev Options</string>
 
+    <string name="today">Today</string>
```

Comment:
@MMurrayAtBR Why are these necessary in the app module?

Reply:
These string resources are being used in the app module "testResources" & "testContext" mocks, but were not in the app module string resources. 
I chose to add the missing strings to the app module res file & resolve the Unresolved References, instead of making changes to the mocks